inspect/scorecalc/pass_fail_calculator.go

```diff
@@ -48,6 +48,16 @@ func (c *chainPassFailCalculator) CalculateScore(results *inspect.InspectionResu
 		return 0, nil
 	}
 
+	// proxy should point to correct chain id
+	if results.Indicators[inspect.IndicatorProxyAPIChainID] != float64(c.config.ChainID) {
+		return 0, nil
+	}
+
+	// trace api should point to correct chain id
+	if results.Indicators[inspect.IndicatorTraceAPIChainID] != float64(c.config.ChainID) {
```

Comment:
Is this safe for chains that don't have trace? (-1 != 56)?

Reply:
great catch, 94dc12e8ada147c812ccdaea8de2c540d7017309 should fix it